spec/javascripts/modules/toggle-class-on-focus.spec.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-env jasmine */
+/* eslint-disable no-multi-str */
+
+describe('A toggle class module', function () {
+  'use strict';
+
+  var element, toggle;
+
+  afterEach(function(){
+    element.remove();
+  });
+
+  describe('when the search box is interacted with', function () {
+    beforeEach(function () {
+      element = $('\
+        <form class="govuk-search" action="/search" method="get" role="search" data-module="toggle-class-on-focus">\
```

Comment:
You could probably make these test fixtures more minimal – ie the minimum markup needed to make this work:

```html
<form data-module="toggle-class-on-focus">
  <input class="js-class-toggle">
</form>
```